.github/workflows/check-action-typing.yml

```diff
@@ -0,0 +1,16 @@
+name: Check Action Typing
+
+on:
+  - push
+  - pull_request
+
```

Comment:
I assume this workflow only needs the `contents:read` permission? We should set it explicitly